Lesson24-signup-form/assets/assets/ezgi/app.js

```diff
@@ -0,0 +1,50 @@
+const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+
+const form = document.getElementById('myForm');
+
+form.addEventListener('submit', function (event) {
+	event.preventDefault();
+	let isMailCorrect = true;
+
+const emailInputError = document.getElementById('emailError');
+const emailInput = document.getElementById('email');
+const emailConfirm=document.getElementById("emailInput");
+const initialPage=document.getElementById("container");
+const confirmPage=document.getElementById("confirm");
+
+
+
+// const placeHolder = emailInput.getAttribute("placeholder");
+const st1 = document.styleSheets[0].cssRules[14]; 
```

Comment:
Don't do like this. Instead, creat a class in CSS file that would style the element as you need. Then apply it in CSS when needed. You can change element's classes by accessing classList property.
E.g.
`element.classList.add('class-that-you-want-to-add');`


https://developer.mozilla.org/en-US/docs/Web/API/Element/classList